frontend/build-configs/koblenz/index.ts

```diff
@@ -34,6 +34,10 @@ export const koblenzCommon: CommonBuildConfigType = {
         showcase: "https://api.entitlementcard.app/project/showcase.entitlementcard.app/map",
         local: "http://localhost:8000/project/koblenz.sozialpass.app/map",
     },
+    mapAttribution: {
+        text: 'Stadt Koblenz',
+        url: 'https://koblenz.de/',
+    },
```

Comment:
:question: Why do we even show that as map attribution? Its not like they created the map or anything, right?

Reply:
Lbe even not but they are responsable for the content